impeller/renderer/backend/vulkan/formats_vk.h

```diff
@@ -160,4 +160,37 @@ constexpr vk::SampleCountFlagBits ToVKSampleCount(SampleCount sample_count) {
   }
 }
 
+constexpr vk::Filter ToVKSamplerMinMagFilter(MinMagFilter filter) {
+  switch (filter) {
+    case MinMagFilter::kNearest:
+      return vk::Filter::eNearest;
+    case MinMagFilter::kLinear:
+      return vk::Filter::eLinear;
+  }
```

Comment:
The pattern is to put an `FML_UNREACHABLE` at the end instead of a default of missing case.

Reply:
Here and elsewhere.

Reply:
done